### PR TITLE
Glossary additions

### LINKED ISSUE #1: words to add to glossary
- [x] fast spiking unit
- [x] regular spiking unit
- [x] waveform
- [x] LFP

### PR DESCRIPTION
**Pull request by galenlynch**
Closes #1

### CODE REVIEW

File: databook/glossary.md [hunk 1]
```diff
@@ -113,6 +139,19 @@ Neurogliaform cell
...
+RS
+  Neurons that, when injected with a long step of current in the context of
+  intracellular recordings, show spike frequency adaptation where the rate of
+  spiking decreases over time. These neurons also have longer (or wider) action
+  potentials, and lower spike rates even when injected with large currents due
+  to hyperpolarization after each action potential. These are the most common
+  type of neurons in the mammalian cortex, and are often associated excitatory
+  neurons. In extracellular recordings, neurons with longer action potentials
+  are also sometimes referred to as regular spiking neurons, a feature which is
+  used to putatively classify them as excitatory neurons
```
Comment (saskiad) [inline databook/glossary.md:152]: VIP as also RS.

File: databook/glossary.md [hunk 2]
```diff
@@ -18,6 +18,18 @@ CSV
 Ephys
   Shorthand for electrophysiology.
 
+Fast spiking neuron
```
Comment (saskiad) [inline databook/glossary.md:21]: unit?

File: databook/glossary.md [hunk 3]
```diff
@@ -18,6 +18,18 @@ CSV
...
+FSN
+FSI
+    Fast spiking neurons are so called because of their "narrow," fast action
+    potentials, specifically as seen in intracellular recordings of a cell in
+    response to a prolonged step of current. Additionally, with sufficient
+    current injection fast spiking neurons exhibit fast spike rates, and do
+    not show frequency adaptation, or slowing of spike rates, over time. In
+    unlabeled extracellular recordings, units with narrow action potentials are
+    also referred to as fast spiking neurons. This feature is sometimes used
+    to putatively label neurons with narrow spikes as {term}`PV+ neuron`s.
```
Comment (saskiad) [inline databook/glossary.md:31]: I really am not comfortable suggesting FS units are exclusively PV neurons.

**Review by saskiad — CHANGES_REQUESTED** [review]
also check the duplicate interneuron entries. I removed one from my PR - which will conflict with this one.

File: databook/glossary.md (hunk 2 above)
Comment (galenlynch) [reply databook/glossary.md]: In the context of intracellular recordings these are neurons, however I said in the context of extracellular recordings "units are" (etc...).

File: databook/glossary.md (hunk 1 above)
Comment (galenlynch) [reply databook/glossary.md]: Added text to emphasize that it's not exclusive

File: databook/glossary.md (hunk 3 above)
Comment (galenlynch) [reply databook/glossary.md]: Added text to emphasize that it's not exclusive

**Comment by galenlynch** [comment]
Pushed a commit to address your PR comments.

**Pull request merged**